jsdoc.json

```diff
@@ -0,0 +1,80 @@
+{
+	"plugins": [ "jsdoc" ],
+	"settings": {
+		"jsdoc": {
+			"tagNamePreference": {
+				"alternateClassName": "alias",
+				"arg": "param",
+				"argument": "param",
+				"augments": "extends",
+				"cfg": "cfg",
+				"chainable": "chainable",
+				"context": "this",
```

Comment:
Do we need to choose one over the other for jsduck back compat?

Both are used across MW, maybe @context slightly more, but @this is the jsdoc standard so I went with that.

Reply:
Sounds good to me. Note that neither `@context` nor `@this` is standard in JSDuck. It's part of our Ruby config file. So updating this will require a one line change to that file as well (no problem).